tests/test_manipulation.py

```diff
@@ -790,6 +790,32 @@ def test_negative_resize(self, xp):
             xp.resize(a, new_shape=new_shape)
 
 
+class TestReshape:
```

Comment:
Do we need to update something in `tests/third_party/cupy/manipulation_tests/test_shape.py`?